packages/core/package.json

```diff
@@ -11,28 +11,42 @@
   "collection:main": "dist/collection/index.js",
   "unpkg": "dist/dso-toolkit/dso-toolkit.js",
   "files": [
-    "dist/",
-    "loader/"
+    "/dist/!(**/*.stories.*)",
+    "/loader"
   ],
   "scripts": {
-    "build": "stencil build --docs",
-    "start": "stencil build --dev --watch --serve",
+    "build": "stencil build --docs-readme",
+    "start": "stencil build --watch --serve --no-open",
     "stencil": "stencil",
-    "generate": "stencil generate"
+    "generate": "stencil generate",
+    "storybook": "start-storybook --static-dir ./dist --port 56006"
```

Comment:
`--static-dir ./dist` zorgt ervoor dat de Stencil compiler build artifacts beschikbaar zijn in het preview iframe van Storybook